CHANGELOG.md

```diff
@@ -22,7 +22,7 @@
    - Adds `vendor-management` as a new delta-stream
  - Add server-specific configurations for `delta-producer-background-jobs-initiator` and `delta-producer-publication-graph-maintainer` (DL-5752)
    - Resolves merge conflict issues when adding a new delta stream or editing an existing one
- - Add `restart: always` for the `virus-scanner` service
```

Comment:
This doesn't really need to be in the changelog, since it's just an internal technical thing.

Reply:
Yes, you're right indeed.